calc.rb

```diff
@@ -0,0 +1,66 @@
+answer = nil
+puts "Hi there"
+while answer == nil 
+  puts "Please enter one of the following operations, addition, subtraction, \nmultiplication, division, exponent, or modulo:"
+  operation = gets.chomp.downcase
+#gets user input
+  puts "Please enter the first number:"
+  user_num1 = gets.chomp
+  num1 = user_num1.to_f
+  
+  puts "Please enter the second number:"
+  user_num2 = gets.chomp
+  num2 = user_num2.to_f
+#test to see if user input is acutally a number
+  if user_num1.to_i == 0 && user_num1 != "0"
+    puts user_num1.to_i
+    puts "You did not enter a number. You entered #{user_num1.class} or #{user_num2.class} type."
+    next
+  elsif user_num2.to_i == 0 && user_num2 != "0"
+    puts "You did not enter a number. You entered #{user_num1.class} or #{user_num2.class} type."
+    next
+
```

Comment:
The segment of code that's executed in this conditional looks very similar to the code above it. You might consider combining it with the one above.